Gemfile

```diff
@@ -101,7 +101,6 @@ group :test do
   gem 'database_cleaner'
   gem 'email_spec'
   gem 'factory_girl_rails'
-  gem 'fakefs', require: 'fakefs/safe'
```

Comment:
Did you mean to remove this gem? It's required by one of the specs, and is causing Circle CI to fail.

Reply:
Hmmm. I didn't touch that line, so I'm not sure where that deletion is coming from. Happy to add it back in, though.